webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+import { TableVariant } from '@patternfly/react-table';
+import { translate as __ } from 'foremanReact/common/I18n';
+import LongDateTime from 'foremanReact/components/common/dates/LongDateTime';
+import { STATUS } from 'foremanReact/constants';
+import PropTypes from 'prop-types';
+
+import TableWrapper from '../../../../components/Table/TableWrapper';
+import { getContentViewHistories } from '../ContentViewDetailActions';
+import {
+  selectCVHistories,
+  selectCVHistoriesStatus,
+  selectCVHistoriesError,
+} from '../ContentViewDetailSelectors';
+
+const ContentViewHistories = ({ cvId }) => {
+  const response = useSelector(state => selectCVHistories(state, cvId), shallowEqual);
+  const status = useSelector(state => selectCVHistoriesStatus(state, cvId), shallowEqual);
+  const error = useSelector(state => selectCVHistoriesError(state, cvId), shallowEqual);
+  const loading = status === STATUS.PENDING;
+
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [searchQuery, updateSearchQuery] = useState('');
+
+  const columnHeaders = [
+    __('Date'),
+    __('Version'),
+    __('Status'),
+    __('Action'),
+    __('Description'),
+    __('User'),
+  ];
+
+  const buildRows = (results) => {
+    const newRows = [];
+    results.forEach((history) => {
+      const {
+        version,
+        created_at: createdAt,
+        status: taskStatus,
+        desciption,
+        action,
+        user,
+      } = history;
+
+      const cells = [
+        { title: <LongDateTime date={createdAt} showRelativeTimeTooltip /> },
+        version,
+        taskStatus,
```

Comment:
Now that I see the table in action I'm realizing it feels strange not to have anything clickable at all.  We could add an `<a>` pretty easily to the status that would link directly to the individual task, since the task ID is already in the API response.  Thoughts?

(Ignore me if this is something we already decided not to do)